lib/msal-node/docs/key-vault-managed-identity.md

```diff
@@ -17,18 +17,10 @@ Now you can import your certificate to Key Vault. **Azure Key Vault** expects ce
 * *.pem* file format contains one or more X509 certificate files.
 * *.pfx* file format is an archive file format for storing several cryptographic objects in a single file i.e. server certificate (issued for your domain), a matching private key, and may optionally include an intermediate CA.
 
-> :bulb: If you don't have any certificates at hand, you can use Azure Key Vault to generate it for you. It will have the additional benefits of assigning partner Certificate Authority and automating certificate rotation. For more information, see [Quickstart: Generate a certificate with Azure Key Vault using the Azure portal](https://docs.microsoft.com/azure/key-vault/certificates/quick-create-portal)
+> :bulb: If you don't have any certificates at hand, you can use Azure Key Vault to generate it for you. It will have the additional benefits of assigning a partner Certificate Authority and automating certificate rotation. For more information, see [Quickstart: Generate a certificate with Azure Key Vault using the Azure portal](https://docs.microsoft.com/azure/key-vault/certificates/quick-create-portal)
```

Comment:
Consider: "you can use Azure Key Vault to generate one for you."

Reply:
done